flink-connectors/flink-connector-base/src/test/java/org/apache/flink/connector/base/source/reader/fetcher/SplitFetcherManagerTest.java

```diff
@@ -70,7 +70,7 @@ void testCloseFetcherWithException() throws Exception {
         reader.setCloseWithException();
         SplitFetcherManager<Object, TestingSourceSplit> fetcherManager =
                 createFetcher("test-split", reader, new Configuration());
-        fetcherManager.close(1000L);
+        fetcherManager.close(30000L);
```

Comment:
I am curious does this add 29 seconds to the build?

Reply:
No, it won't. Normally the close will return fairly quick (a few milliseconds). The additional timeout is just to avoid occasional failures due to slow machines.